packages/preview/codelst/2.0.1/typst.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "codelst"
+version = "2.0.1"
+entrypoint = "codelst.typ"
+authors = ["Jonas Neugebauer"]
+license = "MIT"
+description = "A typst package to render sourcecode."
+repository = "https://github.com/jneug/typst-codelst"
+exclude = ["example.typ", "example.pdf", "manual.pdf", "manual.typ", "tbump.toml"]
+compiler = "0.0.9"
```

Comment:
0.0.9 is not a valid Typst compiler version. should be 0.9.0 or possible 0.11.0 if this has anything specific to 0.11+

Reply:
Yes, this is specific to 0.11.0

Reply:
Thanks for the heads-up. Should be fixed now.